### ISSUE TITLE
[FEATURE] `kill` operation

### ISSUE THREAD

**Issue by brandonbloom** [deref/exo#188]
We have `stop`, but it has a default timeout. We could add a customizable timeout, but it's sometimes useful to instant-kill a process to see how the system behaves, or if you know it is wedged.

Additionally, it may be useful to send arbitrary unix signals to processes.

See docker-compose's support here:

```
$ docker-compose kill --help
Force stop service containers.

Usage: kill [options] [--] [SERVICE...]

Options:
    -s SIGNAL         SIGNAL to send to the container.
                      Default signal is SIGKILL.
```

**Comment by brandonbloom** [comment]
I'm going to implement this quickly because I'm messing with some example that uses SIGHUP and it would be convenient to have this.

**Comment by brandonbloom** [comment]
implemented by #239
